INFO flwr 2023-12-19 16:46:58,778 | app.py:148 | Starting Flower server, config: ServerConfig(num_rounds=400, round_timeout=None)
INFO flwr 2023-12-19 16:46:58,838 | app.py:168 | Flower ECE: gRPC server running (400 rounds), SSL is disabled
INFO flwr 2023-12-19 16:46:58,838 | server.py:86 | Initializing global parameters
INFO flwr 2023-12-19 16:46:58,838 | server.py:273 | Requesting initial parameters from one random client
server booted up. starting clients...
INFO flwr 2023-12-19 16:47:32,056 | grpc.py:50 | Opened insecure gRPC connection (no certificates were passed)
INFO flwr 2023-12-19 16:47:32,068 | server.py:277 | Received initial parameters from one random client
INFO flwr 2023-12-19 16:47:32,069 | server.py:88 | Evaluating initial parameters
DEBUG flwr 2023-12-19 16:47:32,060 | connection.py:39 | ChannelConnectivity.IDLE
DEBUG flwr 2023-12-19 16:47:32,060 | connection.py:39 | ChannelConnectivity.CONNECTING
DEBUG flwr 2023-12-19 16:47:32,063 | connection.py:39 | ChannelConnectivity.READY
INFO flwr 2023-12-19 16:47:32,066 | grpc.py:50 | Opened insecure gRPC connection (no certificates were passed)
DEBUG flwr 2023-12-19 16:47:32,072 | connection.py:39 | ChannelConnectivity.IDLE
DEBUG flwr 2023-12-19 16:47:32,072 | connection.py:39 | ChannelConnectivity.CONNECTING
DEBUG flwr 2023-12-19 16:47:32,075 | connection.py:39 | ChannelConnectivity.READY
INFO flwr 2023-12-19 16:47:32,076 | grpc.py:50 | Opened insecure gRPC connection (no certificates were passed)
DEBUG flwr 2023-12-19 16:47:32,079 | connection.py:39 | ChannelConnectivity.IDLE
DEBUG flwr 2023-12-19 16:47:32,079 | connection.py:39 | ChannelConnectivity.CONNECTING
DEBUG flwr 2023-12-19 16:47:32,083 | connection.py:39 | ChannelConnectivity.READY
INFO flwr 2023-12-19 16:47:32,475 | grpc.py:50 | Opened insecure gRPC connection (no certificates were passed)
DEBUG flwr 2023-12-19 16:47:32,477 | connection.py:39 | ChannelConnectivity.IDLE
DEBUG flwr 2023-12-19 16:47:32,479 | connection.py:39 | ChannelConnectivity.CONNECTING
DEBUG flwr 2023-12-19 16:47:32,484 | connection.py:39 | ChannelConnectivity.READY
INFO flwr 2023-12-19 16:47:32,896 | grpc.py:50 | Opened insecure gRPC connection (no certificates were passed)
DEBUG flwr 2023-12-19 16:47:32,902 | connection.py:39 | ChannelConnectivity.IDLE
DEBUG flwr 2023-12-19 16:47:32,902 | connection.py:39 | ChannelConnectivity.CONNECTING
DEBUG flwr 2023-12-19 16:47:32,906 | connection.py:39 | ChannelConnectivity.READY
INFO flwr 2023-12-19 16:47:32,950 | grpc.py:50 | Opened insecure gRPC connection (no certificates were passed)
DEBUG flwr 2023-12-19 16:47:32,951 | connection.py:39 | ChannelConnectivity.IDLE
DEBUG flwr 2023-12-19 16:47:32,953 | connection.py:39 | ChannelConnectivity.CONNECTING
DEBUG flwr 2023-12-19 16:47:32,957 | connection.py:39 | ChannelConnectivity.READY
INFO flwr 2023-12-19 16:47:33,696 | grpc.py:50 | Opened insecure gRPC connection (no certificates were passed)
INFO flwr 2023-12-19 16:47:33,706 | grpc.py:50 | Opened insecure gRPC connection (no certificates were passed)
DEBUG flwr 2023-12-19 16:47:33,741 | connection.py:39 | ChannelConnectivity.IDLE
DEBUG flwr 2023-12-19 16:47:33,741 | connection.py:39 | ChannelConnectivity.CONNECTING
INFO flwr 2023-12-19 16:47:33,742 | grpc.py:50 | Opened insecure gRPC connection (no certificates were passed)
INFO flwr 2023-12-19 16:47:33,743 | grpc.py:50 | Opened insecure gRPC connection (no certificates were passed)
DEBUG flwr 2023-12-19 16:47:33,744 | connection.py:39 | ChannelConnectivity.READY
DEBUG flwr 2023-12-19 16:47:33,747 | connection.py:39 | ChannelConnectivity.IDLE
DEBUG flwr 2023-12-19 16:47:33,747 | connection.py:39 | ChannelConnectivity.CONNECTING
DEBUG flwr 2023-12-19 16:47:33,748 | connection.py:39 | ChannelConnectivity.IDLE
DEBUG flwr 2023-12-19 16:47:33,748 | connection.py:39 | ChannelConnectivity.IDLE
DEBUG flwr 2023-12-19 16:47:33,748 | connection.py:39 | ChannelConnectivity.CONNECTING
DEBUG flwr 2023-12-19 16:47:33,749 | connection.py:39 | ChannelConnectivity.CONNECTING
DEBUG flwr 2023-12-19 16:47:33,750 | connection.py:39 | ChannelConnectivity.READY
DEBUG flwr 2023-12-19 16:47:33,752 | connection.py:39 | ChannelConnectivity.READY
DEBUG flwr 2023-12-19 16:47:33,755 | connection.py:39 | ChannelConnectivity.READY
INFO flwr 2023-12-19 16:47:34,575 | server.py:91 | initial parameters (loss, other metrics): 2.3035834972381593, {'accuracy': 0.0923}
INFO flwr 2023-12-19 16:47:34,575 | server.py:101 | FL starting
DEBUG flwr 2023-12-19 16:47:34,575 | server.py:218 | fit_round 1: strategy sampled 10 clients (out of 10)
DEBUG flwr 2023-12-19 16:47:41,159 | server.py:232 | fit_round 1 received 10 results and 0 failures
server strategy: ADMM
number of clients: 10
round:      0
Server-side evaluation loss 2.3035834972381593 / accuracy 0.0923
Traceback (most recent call last):
  File "/tudelft.net/staff-bulk/ewi/insy/DBL/swiergarst/RP_FL/run_server.py", line 151, in <module>
    fl.server.start_server(config=fl.server.ServerConfig(num_rounds=NUM_ROUNDS), strategy=strat)
  File "/tudelft.net/staff-bulk/ewi/insy/DBL/swiergarst/envs/flower2en/lib/python3.9/site-packages/flwr/server/app.py", line 176, in start_server
    hist = _fl(
  File "/tudelft.net/staff-bulk/ewi/insy/DBL/swiergarst/envs/flower2en/lib/python3.9/site-packages/flwr/server/app.py", line 217, in _fl
    hist = server.fit(num_rounds=config.num_rounds, timeout=config.round_timeout)
  File "/tudelft.net/staff-bulk/ewi/insy/DBL/swiergarst/envs/flower2en/lib/python3.9/site-packages/flwr/server/server.py", line 106, in fit
    res_fit = self.fit_round(server_round=current_round, timeout=timeout)
  File "/tudelft.net/staff-bulk/ewi/insy/DBL/swiergarst/envs/flower2en/lib/python3.9/site-packages/flwr/server/server.py", line 244, in fit_round
    ] = self.strategy.aggregate_fit(server_round, results, failures)
  File "/tudelft.net/staff-bulk/ewi/insy/DBL/swiergarst/RP_FL/src/ADMM_strategy.py", line 28, in aggregate_fit
    y_local = json.loads(res.metrics["Y"].decode("utf-8"))
KeyError: 'Y'
slurmstepd: error: *** STEP 9202567.1 ON awi03 CANCELLED AT 2023-12-19T18:17:01 DUE TO TIME LIMIT ***
slurmstepd: error: *** JOB 9202567 ON awi03 CANCELLED AT 2023-12-19T18:17:01 DUE TO TIME LIMIT ***
slurmstepd: error: *** STEP 9202567.0 ON awi03 CANCELLED AT 2023-12-19T18:17:01 DUE TO TIME LIMIT ***
srun: Job step aborted: Waiting up to 32 seconds for job step to finish.
srun: Job step aborted: Waiting up to 32 seconds for job step to finish.
srun: error: awi03: tasks 0-5,7-9: Terminated
srun: launch/slurm: _step_signal: Terminating StepId=9202567.1
